File: plugins/Okhttpload/www/okhttpload.js
var exec = require('cordova/exec');

 exports.upload = function(key, value, success, error) {
	exec(success, error, "Okhttpload", "upload", [key, value]);
 };
 exports.download = function(key, defValue, success, error) {
	exec(success, error, "Okhttpload", "download", [key, defValue]);
 };

File: plugins/Okhttpload/src/ios/Okhttpload.swift
import Foundation
@objc(TkyOkhttpload) class Okhttpload : CDVPlugin {
	@objc(upload:)
	func upload (command:CDVInvokedUrlCommand) {
    	//SP数据的key值
    	let key: String = (command.arguments[0] as? String)!
    	//SP数据的value值
    	let value: String = (command.arguments[1] as? String)!
	}
	
	@objc(download:)
	func download (command:CDVInvokedUrlCommand) {
    	//SP数据的key值
    	let key: String = (command.arguments[0] as? String)!
    	//SP数据的默认值
    	let defValue: String = (command.arguments[1] as? String)!
	}
	
}